boards/arm/mimxrt1064_evk/mimxrt1064_evk.dts

```diff
@@ -52,7 +52,7 @@
 		compatible = "pwm-leds";
 
 		green_pwm_led: green_pwm_led {
-			pwms = <&flexpwm2_pwm3 0>;
+			pwms = <&flexpwm2_pwm3 0 PWM_MSEC(20)>;
```

Comment:
I think you need to do the same in
https://github.com/zephyrproject-rtos/zephyr/blob/8aece303cac08ff6d6c82cd25e2b14b7f827e9ea/boards/arm/mimxrt1160_evk/mimxrt1160_evk.dtsi#L35
and
https://github.com/zephyrproject-rtos/zephyr/blob/8aece303cac08ff6d6c82cd25e2b14b7f827e9ea/boards/arm/mimxrt1170_evk/mimxrt1170_evk.dtsi#L35

Reply:
done